### DISCUSSION TITLE
Select audio input in windows

### DISCUSSION THREAD

**Discussion by lowepalma** [savonet/liquidsoap#3536] [User Support]
In Liquidsoap, I'm setting up a live stream and need to select an audio input in WINDOWS 11. How can this be done?

Best regards,

**Comment by toots** [comment]
Hi @lowepalma. On windows, audio input is provided via portaudio. You can use the `--list-portaudio-devices` command line option to list all available devices:
```shell
% liquidsoap.exe --list-portaudio-devices
```
